src/DependencyInjection/OpstalentCrudExtension.php

```diff
@@ -4,8 +4,8 @@
 
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
-use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
 use Symfony\Component\DependencyInjection\Extension\Extension;
+use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
```

Comment:
IMO there is no sense in changing such things if there is no other change in file